charter.md

```diff
@@ -2,26 +2,24 @@
 
 Modern messaging applications commonly support numerous features including plain and rich text, delivery notifications, read receipts, replies, reactions, and many more. The working group will identify a baseline set of messaging features and specify a content format to allow this feature set to be implemented interoperably. This format must be usable in the presence of E2EE. In defining the format, the working group will seek to reuse existing primitives (especially existing semantics) including previously defined message headers, MIME types, and URIs where practical.
 
-Messaging applications commonly also support voice and video calling and conferencing. The working group will prioritize deliverables needed to support messaging, but may also specify a basic mechanism needed to bootstrap interoperable audio and video connections. The working group will not standardize new signaling or media protocols but may recommend the use of existing protocols and suites such as SIP and WebRTC.
+In its initial phase, the working group will focus on solutions for messaging. The working group will aim for general-purpose designs fit for both 1:1 and multiparty messaging. In a future phase, the working group may recharter to work on audio and video. The working group will not standardize new signaling or media protocols but may recommend the use of existing protocols and suites such as SIP and WebRTC. 
 
-E2EE messaging services rely on metadata processing to manage spam and abuse. The working group will specify general-purpose means for the exchange of such metadata, which may include the signaling of feedback about users/accounts, outputs derived from reputation services, or other information interoperating providers use for filtering purposes. 
-
-For all of its core deliverables, the working group will aim for general-purpose designs, but will prioritize solutions for messaging before audio and video, and will prioritize solutions for one-to-one communications before multiparty communications.
+A recharter would be required should the working group decide to work on:
+* Metadata processing to manage spam and abuse
+* Interoperable mechanisms for group administration or moderation across systems
 
 The following are out of scope for the working group:
 
 * Extensions to the MLS protocol. If needed, requirements will be referred to the MLS working group or other relevant working groups in the security area.
 * Definition of completely new identity formats or protocols.
-* Discovery of which service or services are associated with a recipient's specific phone number or email address without the consent of the recipient.
```

Comment:
Please leave this line in place. To keep us out of an intractable tar pit, we need to make clear that figuring out which service to search is forever out of scope.